Apps/WebClient/src/NewClientApp/src/stores/vaccinationStatusAuthenticated.ts

```diff
@@ -0,0 +1,387 @@
+import { ActionType } from "@/constants/actionType";
+import { ErrorSourceType, ErrorType } from "@/constants/errorType";
+import { ResultType } from "@/constants/resulttype";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import CovidVaccineRecord from "@/models/covidVaccineRecord";
+import { DateWrapper } from "@/models/dateWrapper";
+import { ResultError } from "@/models/errors";
+import { LoadStatus } from "@/models/storeOperations";
+import VaccinationStatus from "@/models/vaccinationStatus";
+import VaccineRecordState from "@/models/vaccineRecordState";
+import { ILogger, IVaccinationStatusService } from "@/services/interfaces";
+import { useErrorStore } from "@/stores/error";
+import { defineStore } from "pinia";
+import { computed, ref } from "vue";
+
+export const useVaccinationStatusStore = defineStore(
+    "vaccinationStatusAuthenticated",
+    () => {
+        const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+        const vaccinationStatusService =
+            container.get<IVaccinationStatusService>(
+                SERVICE_IDENTIFIER.VaccinationStatusService
+            );
+
+        const errorStore = useErrorStore();
+
+        const defaultVaccineRecordState: VaccineRecordState = {
+            hdid: "",
+            download: false,
+            status: LoadStatus.NONE,
+            statusMessage: "",
+            resultMessage: "",
+        };
+
+        interface AuthenticatedVaccinationStatus {
+            vaccinationStatus?: VaccinationStatus;
```

Comment:
Suggest renaming `vaccinationStatus` property to `data` to match other stores.

Reply:
Grouped under computed.

Reply:
Changed.